app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -38,6 +40,7 @@
 import android.widget.ListView;
 import android.widget.TextView;
 
+import net.gsantner.markor.ui.hleditor.Highlighter;
```

Comment:
No Markor references in opoc. Especially this one as it is used for all my other apps as well.

Reply:
Fixed it will a callback instead